metadata-db-app/int_test/cmr/metadata_db/int_test/utility.clj

```diff
@@ -323,6 +330,27 @@
                            (dissoc attributes :extra-fields))]
      (concept provider-id :service uniq-num attributes))))
 
+(defn service-association-concept
+  "Creates a service association concept"
+  ([assoc-concept service uniq-num]
```

Comment:
indentation looks off here and in the arity below

Reply:
Done.